plans/academic_onramp.markdown

```diff
@@ -0,0 +1,122 @@
+three individuals unable to proceed past module 1.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+I do not know.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now? 
+
+I would like whoever is teaching Module 1 to be more active in the preparation of the incoming cohort. I'd like to schedule short 
+meetings each to gauge where they are and how they feel.
+
+* What will be the next three things you do?
+
+I would give 1507 access to Slack right now. I would write a series of diagnostics for 1507 so that we have a better idea of where
+the group stands coming into module one and not say after their first project. I would assign them to posses right now so they meet
+their posse-mates and their mentors.
+
+* How will you keep track of progress and results?
+
+The aforementioned diagnostics from the previous question. We would log those as we currently do. We could also measure their 
+performance after the first project. 
+
+* Are you empowered to take the actions you think necessary?
+
+This is a tricky question, because I don't feel that I should just be making decisions all willy-nilly. That being said, I do feel 
+very comfortable asking the team for their opinions and input and know that any suggestions that I do make would be considered 
+thoughtfully, and I would receive good, specific feedback on my suggestions at the minimum.
+
+ 
+## The Future
+
+* What would total success look like?
+
+The bottom range of project evaluation scores would rise, and the top end as well. 
+
+* What metrics can be used to evaluate results?
+
+Project and assessment scores. The sentiment of end of week surveys and group retros.
+
+* What would meaningful progress look like in 3 months? 6 months? 12 months?
+
+Meaningful progress would show no repeats during module one and fewer people doing The Chisel. Success over longer periods of times 
+would show a general upwards trend.
+
+* How feasible are each of those goals?
+
+Improvement is feasible. Perfection less so. It can be difficult to sustain improvement, as we can only really iterate once every six 
+weeks. If something doesn't work, we have to wait six weeks to try something different. 
+
+
+## The Periphery
+
+* What challenges are you going to stop working on to put time into this?
+
+This shouldn't take up too much of my time, and to start, most of the investment time-wise is up-front. Diagnostics need only 
+be written from scratch once, and are something that can be collaborated on.
+
+
+* What research/learning do you need to do to be effective at solving it?
+
+I think I'd have to take a look and see what new tools there are for learning Ruby or OO or other topics. Pragmatic Studio is 
+fantastic, but I think we should be open to the possibility that there is better.
```

Comment:
I think Pragmatic can be overwhelming for some people. I'm always curious about how many people actually complete the course, and out of those, how many people truly understand what's being taught.